app/controllers/package_category.js

```diff
@@ -31,12 +33,18 @@ export default Ember.Controller.extend({
     ];
   }),
 
+  isCategorySelected: Ember.computed("selectedCategoryId", function() {
+    return !!this.get("selectedCategoryId.id");
```

Comment:
Reason for this `!!` ?

Reply:
![Screen Shot 2019-07-10 at 5 26 58 PM](https://user-images.githubusercontent.com/26408955/60967216-c0854900-a337-11e9-8666-89148c015173.png)
converting it to boolean.

Reply:
You can use bool computed property for this.
https://api.emberjs.com/ember/2.15/namespaces/Ember.computed/methods?anchor=bool

Reply:
I guess the bool property would be the more "ember appropriate" way of doing it. Although `!!` kinda makes us cooler. Tough call 😄 (either way is fine by me)